test/tools/libtesteth/ImportTest.cpp

```diff
@@ -547,23 +547,34 @@ void parseJsonIntValueIntoVector(json_spirit::mValue const& _json, vector<int>&
 		_out.push_back(_json.get_int());
 }
 
-void ImportTest::checkAllowedNetwork(std::set<std::string> const& _networks)
+set<string> const& getAllowedNetworks()
 {
-    set<eth::Network> const& allnetworks = test::getNetworks();
-    set<string> allowedNetowks;
-    allowedNetowks.emplace("ALL");
-    for (auto const& net : allnetworks)
-        allowedNetowks.emplace(test::netIdToString(net));
+    static set<string> allowedNetowks;
+    if (allowedNetowks.empty())
+    {
+        allowedNetowks.emplace("ALL");
```

Comment:
Should this allow `ALL`?

Reply:
this is a list of strings that are allowed at the expect section network field

Reply:
Is `<Homestead` not allowed, but `ALL` is allowed?

Reply:
Same as above.  Consideration for the future.

Reply:
<Homestead not allowed. only Homestead, Frontier, EIP150, EIP158, Byzantium and Constantinople. 
ALL is allowed and means all of that.